docs/tutorials/getting-started-with-csharp/microservices.md

```diff
@@ -0,0 +1,443 @@
+file contains information about each package that is referenced. After restoring
+all the dependencies, you build the application:
+
+`dnu build`
+
+And once you build the application, you run it from the command line:
+
+`dnx web`
+
+The default configuration listens to http://localhost:5000. You can open a
+browser and navigate to that page and see a "Hello World!" message.
+
+## Anatomy of an ASP.NET Core application
+
+Now that you've built the application, let's look at how this functionality
+is implemented. There are two of the generated files that are particularly
+interesting at this point: project.json and startup.cs. 
+
+Project.json contains information about the project. The three nodes you'll
+often work with are 'dependencies', 'commands' and 'frameworks'. The
+dependencies node lists all the packages that are needed for this application.
+At the moment, this is a small node, needing only the packages that run the
+web server.
+
+The 'commands' node lists the command that runs the web server. As a project
+grows, you'll add commands here to run unit tests, or perform other tasks. The
+web task starts the webserver.
+
+The 'frameworks' node specifies the versions and configurations of the .NET
+framework that will run this application.
+
+The application is implemented in Startup.cs. This file contains the startup
+class. It's `Main()` method starts the web server, and instructs the web
+server that the web application class is this Startup class:
+
+`public static void Main(string[] args) => Microsoft.AspNet.Hosting.WebApplication.Run<Startup>(args);`
+
+This method is a simple one-line method, so it utilizes the C# syntax for
+*expression bodied members*. The body of the method, instead of being enclosed
+in curly braces, is represented by the body of a lambda expression.
+
+The other two methods are called by the asp.net core infrastructure to configure
+and run the applicatioh. The `ConfigureServices` method describes the services that are
+necessary for this application. You're building a lean microservice, so it doesn't
+need to configure any dependencies. The `Configure` method configures the handlers
+for incoming HTTP Requests. The template generates a simple handler that responds
+to any request with the text 'Hello World!'.
+
+# Build a microservice
+
+The service you're going to build will deliver weather reports from anywhere
+around the globe. We'll save costs on the implementation by not doing any
+actual weather forecasting. Weather forecasts aren't always accurate.
+Let's assume random data is just as good. 
+
+There are a number of tasks you'll need to perform in order to implement
+our random weather service:
+
+* Parse the incoming request to read the latitude and longitude.
+* Generate some random forecast data.
+* Convert that random forecast data from C# objects into JSON packets.
+* Set the response header to indicate that your service sends back JSON.
+* Write the response.
+
+Let's get started.
```

Comment:
Again, don't use "Let's". We're talking to the reader, not working alongside them :-)